### PR TITLE
Updating the upgrade process document

### PR DESCRIPTION
**Pull request by rksakamoto**
Including the v2.0 special procedures for ACP and APIM components

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

File: content/en/docs/deployment/upgrade.md [hunk 1]
```diff
@@ -1,60 +1,80 @@
...
+
+## API Managemente component prerequisites
+
+If you are migrating from version 1.3 to 1.4, the API management component has been split into two Helm charts: `apim` for products and `apim-config` for configuration. In this case you must therefore uninstall `apim`, verify it has been completely removed, and then install `apim` and `apim-config`.
+
+If you are upgrading from old version of API management component (before apim helmchart 1.7 ), it will be ncessary to upgrade the  cassandra database from v2.x to 3.11 before upgrade the APIM package - please follow the <https://docs.axway.com/bundle/axway-open-docs/page/docs/apim_installation/apigw_upgrade/upgrade_cassandra/index.html>  document.
+
+## ACP component prerequisite
+
+If you are upgrading the ACP components from previous releases previous than helmchart 2.0 , it is necessary to update the cockroachdb to v21.2 before the upgrade of acp package:
```
Comment (sallylanda6) [inline content/en/docs/deployment/upgrade.md:17]:
```suggestion
If you are upgrading the ACP components from previous releases before Helm chart v2.0, it is necessary to update the CockroachDB to v21.2 before you upgrade the acp package:

```

Comment (sallylanda6) [reply content/en/docs/deployment/upgrade.md:17]: The Installation topic refers to the ACP components (ACP and Consent) as Cloud Entity installation. Will users know that ACP components is Cloud Entity?

File: content/en/docs/deployment/upgrade.md [hunk 2]
```diff
@@ -1,60 +1,80 @@
...
+helm upgrade -n open-banking-acp acp acp/kube-acp-stack -f open-banking-acp/files/acp.values.yaml --version 0.15.3
+```
+
+Wait for all pods to be running and now uninstall the acp-prereq.
+
+```bash
+helm uninstall [ACP_PREREQ_RELEASE_NAME] -n [NAMESPACE]
+```
+
+After this it is ready to upgrade process.
```
Comment (sallylanda6) [inline content/en/docs/deployment/upgrade.md:29]:
```suggestion
```

Comment (sallylanda6) [reply content/en/docs/deployment/upgrade.md:29]: I don't think it's necessary to include that sentence.

**Review by sallylanda6 — CHANGES_REQUESTED** [review]
I completed a copy edit for consistency and standards. I also added one question that may or may not need any changes.

File: content/en/docs/deployment/upgrade.md [hunk 3]
```diff
@@ -1,60 +1,80 @@
...
+title: "Upgrade"
+linkTitle: "Upgrade"
+weight: 4
+---
+
+If a new Open Banking API is released or an existing Open Banking API is updated you can perform an upgrade to the deployed application.
+
+## API Managemente component prerequisites
+
+If you are migrating from version 1.3 to 1.4, the API management component has been split into two Helm charts: `apim` for products and `apim-config` for configuration. In this case you must therefore uninstall `apim`, verify it has been completely removed, and then install `apim` and `apim-config`.
```
[the remark this answers is not in the document]
Comment (sallylanda6) [reply content/en/docs/deployment/upgrade.md]: Oops, I edited this incorrectly. I intended to delete the word "therefore" and not "uninstall." Please add "uninstall" back.

**Review by rksakamoto — COMMENTED** [review]
ok

**Review by sallylanda6 — APPROVED** [review]

**Pull request merged**